src/main/webapp/WEB-INF/tags/instructor/results/copyPasteModal.tag

```diff
@@ -17,11 +17,10 @@
   <div class="modal-dialog modal-lg">
     <div class="modal-content">
       <div class="modal-header">
-        <span class="help-block" style="display:inline;">
-          Tips: After selecting the table, <kbd>Ctrl + C</kbd> to COPY and
-          <kbd>Ctrl + V</kbd> to PASTE to your Excel Workbook.
+        <span class="help-block">
+          Tips: After selecting the table, <kbd>Ctrl + C</kbd> or <kbd>&#8984; + C</kbd> to COPY and
+          <kbd>Ctrl + V</kbd> or <kbd>&#8984; + V</kbd> to PASTE to your Excel Workbook.
         </span>
-        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
This is an unrelated change.

Reply:
The reason I removed this is because it seemed extraneous and it serves no purpose now that the tip is displayed as a block with the buttons below it. 

This seems to have been used to ensure that there was some space between the end of the tip text and the buttons when they were on the same line.

Reply:
Apologies; I didn't realise the change in UI (the tip now being displayed as a block). This is fine then.